build/OutputDirectories.props

```diff
@@ -25,5 +25,15 @@
       <TestPackagesDir>$(TestOutputDir)/packages/</TestPackagesDir>
       <TestArtifactsDir>$(TestOutputDir)/artifacts/</TestArtifactsDir>
       <TestResultXmlDir>$(TestOutputDir)/results/</TestResultXmlDir>
+      <ArchiveOutputDirectory>$(PackagesDirectory)</ArchiveOutputDirectory>
```

Comment:
I checked with @piotrpMSFT about this. First we'll go ahead and finish the work to get properties out of targets. Then take a second pass to clean up stuff.